pkg/proxy/integrations/httpparser/httpparser.go

```diff
@@ -31,17 +32,17 @@ type HttpParser struct {
 }
 
 // ProcessOutgoing implements proxy.DepInterface.
-func (http *HttpParser) ProcessOutgoing(request []byte, clientConn, destConn net.Conn, ctx context.Context) {
+func (http *HttpParser) ProcessOutgoing(request []byte, clientConn, destConn net.Conn, ctx context.Context, sourcePort int) {
```

Comment:
Can't you extract port from clientConn or destConn..?

Reply:
will increase computation time, better to use a precalculated value